homeassistant/components/homewizard_climate/const.py

```diff
@@ -0,0 +1,5 @@
+"""Constants for the Homewizard Climate integration."""
+
+DOMAIN = "homewizard_climate"
+USERNAME = "username"
+PASSWORD = "password"
```

Comment:
These can be imported from `homeassistant.const` as `CONF_USERNAME` and `CONF_PASSWORD`. I think you can remove it here and import these in `__init__.py` and `config_flow.py` directly.

Reply:
Thanks for the callout, done.